src/commands/paint.ts

```diff
@@ -47,22 +47,22 @@ export async function getPort(defaultPort: number): Promise<number> {
   return bestAvailablePort;
 }
 
-function getStateString(workerState: any, isWatch: boolean): [chalk.ChalkFunction, string] {
+function getStateString(workerState: any, isWatch: boolean): [colors.Colorize, string] {
```

Comment:
Nice :) I'm glad that type came in handy for someone else haha

Reply:
So since I couldn't get the typings provided to work, this actually just results in `any` currently 😢

Reply:
Interesting. Is there something special about the TS config here? It works for me in multiple places

Reply:
Do you have an example I can take a look at?

Also, @FredKSchott do you have any idea why it might not be picking up the typing?

Reply:
Ah, funny enough, I was getting the type information from within JS files – not TS! 😆 https://github.com/microsoft/TypeScript/issues/33079#issuecomment-647051147

Reply:
Interesting... 🤔 Thanks for looking into it!